store/tikv/gcworker/gc_worker.go

```diff
@@ -330,26 +330,33 @@ func (w *GCWorker) checkPrepare(ctx context.Context) (bool, uint64, error) {
 }
 
 // calculateNewSafePoint uses the current global transaction min start timestamp to calculate the new safe point.
-func (w *GCWorker) calSafePointByMinStartTS(safePoint time.Time) time.Time {
+func (w *GCWorker) calSafePointByMinStartTS(ctx context.Context, safePoint time.Time) time.Time {
 	kvs, err := w.store.GetSafePointKV().GetWithPrefix(infosync.ServerMinStartTSPath)
 	if err != nil {
-		logutil.BgLogger().Warn("get all minStartTS failed", zap.Error(err))
+		logutil.Logger(ctx).Warn("get all minStartTS failed", zap.Error(err))
 		return safePoint
 	}
 
-	safePointTS := variable.GoTimeToTS(safePoint)
+	var globalMinStartTS uint64 = math.MaxUint64
```

Comment:
Why change it?

Reply:
To make it possible to print the log conditionally at L353

Reply:
ok